server/views/pages/create/initialMeetingTime.njk

```diff
@@ -98,7 +98,7 @@
                     {{ govukRadios({
                     id: "appointment-time-type",
                     name: "appointmentTimeType",
-                    value: formResponses.appointmentTimeType or licence.appointmentTimeType,
+                    value: (formResponses.appointmentTimeType or licence.appointmentTimeType) or SPECIFIC_DATE_TIME,
```

Comment:
To select Date time option default when the user first comes to this screen.

Reply:
is this needed to expand the box?

Reply:
Yes it is needed.